src/components/Header/TopNav/outerItemStyles.js

```diff
@@ -0,0 +1,68 @@
+import { css } from 'styled-components'
+import remcalc from 'remcalc'
```

Comment:
so many styles 🤯

Reply:
You talking about us having the `remcalc` library? Or just that we have so many styles files? :laughing: 
:boom:

Reply:
so many js files 😄 I think that if it were repeated css classes, we'd not see it so badly but as we see so many files being exported/imported, it feels a bit scary haha

Reply:
Yep. It's true. I feel like it's just a weird, relatively unusual quirk of needing to re-use styles across different components - e.g. DropdownName and OuterListItem.

Reply:
yeah, this is an unusual case. But I'm thinking to myself that if we were using "traditional" css,  we'd just apply some common classes (that would represent these common appearances) to these elements.. and that's essentially what we're doing by passing these object styles (with a bit overload of more css being outputed, I guess)